Chapters/FirstContact/FirstContactShrunk.md

```diff
@@ -134,10 +132,10 @@ CustomerSatisfactionPresenter >> defaultLayout
 ```
 
 
-In this layout, we add two rows to the UI, one with the buttons and one with the screen of text. Defining widget layout is a complex process with many different possible requirements, hence in this chapter we do not talk in detail about layout specification. For more information we refer to Chapter *@cha_layout_construction@*.
+In this layout, we add two rows to the UI, one with the buttons and one with the screen of text. Defining widget layout is a complex process with many different possible requirements, hence in this chapter we do not talk in detail about layout specification. For more information we refer to Chapter *@cha_layout@*.
 
 
-Once the class method `defaultLayout` is defined, you can start to open your UI as follows: `CustomerSatisfactionPresenter new open`. You should obtain a widget similar to the one shown in Figure *@figFirstCut@*.
+Once the method `defaultLayout` is defined, you can start to open your UI as follows: `CustomerSatisfactionPresenter new open`. You should see a window similar to the one shown in Figure *@figFirstCut@*.
```

Comment:
The figure shows a window, not a single widget.

Reply:
I like your review. I do not see this kind of mistakes.